src/app/core/workbox/workbox.component.ts

```diff
@@ -49,7 +54,13 @@ import {ClosingDirtyAppsModalComponent} from "../modals/closing-dirty-apps/closi
                                 <i class="fa fa-file-text-o"></i>
                             </div>
 
-                            <div class="title" [ct-tooltip]="ctt" [tooltipPlacement]="'bottom'">{{tab.label}}</div>
+                            <div class="title"
+                                 [ct-tooltip]="ctt" 
+                                 [tooltipPlacement]="'bottom'" 
+                                 data-test="file-tab-title"
+                                 [attr.data-label]="tab.label">
```

Comment:
It's not necessary to have a data-test attribute that's the same content as the text of the element.